schemes/sima_diagnostics/rayleigh_friction_diagnostics.F90

```diff
@@ -0,0 +1,64 @@
+module rayleigh_friction_diagnostics
+
+   use ccpp_kinds, only:  kind_phys
+
+   implicit none
+   private
+   save
+
+   public :: rayleigh_friction_diagnostics_init ! init routine
+   public :: rayleigh_friction_diagnostics_run  ! main routine
+
+CONTAINS
+
+   !> \section arg_table_rayleigh_friction_diagnostics_init  Argument Table
+   !! \htmlinclude rayleigh_friction_diagnostics_init.html
+   subroutine rayleigh_friction_diagnostics_init(errmsg, errflg)
+      use cam_history,         only: history_add_field
+      use cam_history_support, only: horiz_only
+
+      character(len=512), intent(out) :: errmsg
+      integer,            intent(out) :: errflg
+
+      ! Local variables:
+
+      errmsg = ''
+      errflg = 0
+
+      ! History add field calls
+      call history_add_field('UTEND_RAYLEIGH', 'Zonal wind tendency due to Rayleigh Friction', 'lev', 'avg', 'm s-2')
+      call history_add_field('VTEND_RAYLEIGH', 'Meridional wind tendency due to Rayleigh Friction', 'lev', 'avg', 'm s-2')
+      call history_add_field('STEND_RAYLEIGH', 'Static energy tendency due to Rayleigh Friction', 'lev', 'avg', 'J kg-1')
+      
+   end subroutine rayleigh_friction_diagnostics_init
+
+   !> \section arg_table_rayleigh_friction_diagnostics_run  Argument Table
+   !! \htmlinclude rayleigh_friction_diagnostics_run.html
+   subroutine rayleigh_friction_diagnostics_run(dudt, dvdt, dsdt, errmsg, errflg)
+
+      use cam_history, only: history_out_field
+      !------------------------------------------------
+      !   Input / output parameters
+      !------------------------------------------------
+      ! State variables
+      real(kind_phys), intent(in) :: dudt(:,:) !tendency_of_x_wind due to RF
+      real(kind_phys), intent(in) :: dvdt(:,:) !tendency_of_y_wind due to RF
+      real(kind_phys), intent(in) :: dsdt(:,:) !tendency_of_y_wind due to RF
```

Comment:
I would update these variable comments to the following:

```suggestion
      real(kind_phys), intent(in) :: dudt(:,:) !tendency_of_eastward_wind due to RF
      real(kind_phys), intent(in) :: dvdt(:,:) !tendency_of_northward_wind due to RF
      real(kind_phys), intent(in) :: dsdt(:,:) !tendency_of_dry_air_enthalpy_at_constant_pressure due to RF
```

Reply:
Good eye, good eye. Fixed